core/src/main/java/org/elasticsearch/plugins/PluginManager.java

```diff
@@ -101,6 +101,7 @@
             "discovery-ec2",
             "discovery-gce",
             "discovery-multicast",
+            "ingest-geoip",
```

Comment:
I think I added all the missing bits to PluginManager, but we have to remember to test the installation of this one through download.elastic.co , not sure when is a good time for it.